setup.cfg

```diff
@@ -137,52 +137,43 @@ skip=
-    pandas/tests/io/formats/test_to_csv.py,
-    pandas/tests/io/formats/test_css.py,
-    pandas/tests/io/formats/test_to_latex.py,
-    pandas/tests/io/formats/test_printing.py,
-    pandas/tests/io/msgpack/test_buffer.py,
-    pandas/tests/io/msgpack/test_read_size.py,
-    pandas/tests/io/msgpack/test_pack.py,
-    pandas/tests/io/msgpack/test_except.py,
-    pandas/tests/io/msgpack/test_unpack_raw.py,
-    pandas/tests/io/msgpack/test_unpack.py,
-    pandas/tests/io/msgpack/test_newspec.py,
-    pandas/tests/io/msgpack/common.py,
-    pandas/tests/io/msgpack/test_limits.py,
-    pandas/tests/io/msgpack/test_extension.py,
-    pandas/tests/io/msgpack/test_sequnpack.py,
-    pandas/tests/io/msgpack/test_subtype.py,
-    pandas/tests/io/msgpack/test_seq.py,
-    pandas/tests/io/json/test_compression.py,
-    pandas/tests/io/json/test_ujson.py,
-    pandas/tests/io/json/test_normalize.py,
-    pandas/tests/io/json/test_readlines.py,
-    pandas/tests/io/json/test_pandas.py,
-    pandas/tests/io/json/test_json_table_schema.py,
+    pandas/tests/indexes/test_frozen.py,
+    pandas/tests/indexes/test_base.py,
+    pandas/tests/indexes/test_category.py,
+    pandas/tests/indexes/datetimes/test_missing.py,
+    pandas/tests/indexes/multi/test_duplicates.py,
+    pandas/tests/indexes/multi/test_partial_indexing.py,
+    pandas/tests/indexes/multi/test_names.py,
+    pandas/tests/indexes/multi/test_reshape.py,
+    pandas/tests/indexes/multi/test_format.py,
+    pandas/tests/indexes/multi/test_set_ops.py,
+    pandas/tests/indexes/multi/test_monotonic.py,
+    pandas/tests/indexes/multi/test_reindex.py,
+    pandas/tests/indexes/multi/test_drop.py,
+    pandas/tests/indexes/multi/test_integrity.py,
+    pandas/tests/indexes/multi/test_astype.py,
+    pandas/tests/indexes/multi/test_analytics.py,
+    pandas/tests/indexes/multi/test_missing.py,
+    pandas/tests/indexes/multi/conftest.py,
+    pandas/tests/indexes/multi/test_join.py,
+    pandas/tests/indexes/multi/test_conversion.py,
+    pandas/tests/indexes/interval/test_construction.py,
+    pandas/tests/indexes/interval/test_interval_new.py,
+    pandas/tests/indexes/interval/test_interval.py,
+    pandas/tests/indexes/interval/test_interval_range.py,
+    pandas/tests/indexes/interval/test_astype.py,
+    pandas/tests/indexes/interval/test_interval_tree.py,
+    pandas/tests/indexes/timedeltas/test_indexing.py,
+    pandas/tests/indexes/timedeltas/test_construction.py,
+    pandas/tests/indexes/timedeltas/test_setops.py,
+    pandas/tests/indexes/timedeltas/test_timedelta.py,
+    pandas/tests/indexes/timedeltas/test_tools.py,
+    pandas/tests/indexes/timedeltas/test_arithmetic.py,
+    pandas/tests/indexes/timedeltas/test_astype.py,
+    pandas/tests/indexes/timedeltas/test_scalar_compat.py,
+    pandas/tests/indexes/timedeltas/test_partial_slicing.py,
+    pandas/tests/indexes/timedeltas/test_timedelta_range.py,
+    pandas/tests/indexes/timedeltas/test_ops.py,
```

Comment:
why are we adding all these back? they should be ok if they are not in the exclude list, right?